stdlib/3/io.pyi

```diff
@@ -77,48 +79,15 @@ class FileIO(RawIOBase):
         opener: Optional[Callable[[Union[int, str], str], int]] = ...
     ) -> None: ...
 
-# TODO should extend from BufferedIOBase
-class BytesIO(BinaryIO):
+class BytesIO(BufferedIOBase, IO[bytes]):
```

Comment:
Does inheriting from `BinaryIO` instead of `IO[bytes]` cause any trouble? There may well be code relying on BytesIO being assignable to BinaryIO.

Reply:
It seems that is fine, changed.